.github/workflows/pre-merge-ci.yml

```diff
@@ -3,8 +3,13 @@ name: Pre-Merge Integration tests
 on: 
   pull_request:
     types: [ labeled, opened, synchronize, reopened ]
+    paths:
+      - 'src/golang/**'
+      - 'src/python/**'
+      - 'integration_tests/**'
+      - 'sdk/aqueduct/**'
 
 jobs:
   trigger-integration-tests:
-    uses: aqueducthq/aqueduct/.github/workflows/integration-tests.yml@main
-    if: contains(github.event.pull_request.labels.*.name, 'run_integration_test')
\ No newline at end of file
+    if: contains(github.event.pull_request.labels.*.name, 'run_integration_test')
```

Comment:
so the test will be triggered if both the content in `paths` are changed and the label exists?

Reply:
Yes, that's correct!